lib/xander/query/response.ex

```diff
@@ -4,14 +4,19 @@ defmodule Xander.Query.Response do
   @message_response 4
 
   def parse_response(full_response) do
-    <<_header_todo_investigate::binary-size(8), response_payload::binary>> = full_response
+    %{payload: response_payload} = Xander.Util.plex(full_response)
 
     case CBOR.decode(response_payload) do
       {:ok, decoded, ""} -> parse_cbor(decoded)
       {:error, _reason} -> {:error, :error_decoding_cbor}
     end
   end
 
+  # For get_current_block_height
+  defp parse_cbor([@message_response, [1, response]]) do
```

Comment:
I couldn't figure out what the `1` is for. I'd make a named module level variable for it if I did.

Reply:
It looks like this number 1 represents the `slot_timeline`:

https://github.com/txpipe/pallas/blob/main/pallas-network/src/miniprotocols/localstate/queries_v16/mod.rs#L100-L101

Found this on the following PR that adds the `GetChainBlockNo` query to pallas:

https://github.com/txpipe/pallas/pull/441/files

Reply:
I poked around some, but couldn't find anymore info on slot_timeline to annotate the variable with. This doc suggests that it's a fixed value, so based on that it seems safe to pattern match on the 1.

https://arsmagna.xyz/docs/network-lsq/#available-queries-and-results